data/korath/korath outfits.txt

```diff
@@ -105,7 +131,7 @@ outfit "Systems Core (Small)"
 	"hull repair rate" 0.2
 	"hull energy" 0.2
 	"hull heat" 0.3
-	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small number of "hull-walking" welding robots that can make minor repairs to the ship's outer armor while in flight.`
+	description `The heart of any Korath ship is the Systems Core, which houses the ship's central computer, shield generators, and a small amount of hull-welding microbots that can make minor repairs to the ship's outer armor while in flight.`
```

Comment:
As someone who dabbles in metalwork I get that welding sounds too specific :-)

Walking sounds clumsy to me though.  Swarming, as you put it, is evocative, visually stimulating, and interesting, walking sounds like a clumsy thing.  Maybe the Coal repair bots walk, because we can actually see they're quite big fellows, but walking doesn't sound like the right verb to me for microbots.

Not opposed to removing the stipulation about "minor repairs to the hull," though if you want to be honest, that's all that ever gets damaged on your ship in ES.

Reply:
Sure, swapping out swarming for walking sounds good.